src/models/generation/hydro_generation.jl

```diff
@@ -23,7 +23,7 @@ TechHydro(;installedcapacity = 0.0,
           realpowerlimits = @NT(min = 0.0, max = 0.0),
           reactivepower = nothing,
           reactivepowerlimits = nothing,
-          ramplimits = nothing,
+          ramplimits = @NT(up=0.0, down=0.0),
```

Comment:
Why are you adding default 0.0 ramps?

Reply:
I can change to 10.0 , it just needs to be something instead of a Nothing